libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/InstructionView.java

```diff
@@ -446,9 +450,9 @@ private void bind() {
   private void initializeBackground() {
     if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
       int navigationViewBannerBackgroundColor = ThemeSwitcher.retrieveThemeColor(getContext(),
-              R.attr.navigationViewBannerBackground);
+        R.attr.navigationViewBannerBackground);
```

Comment:
For Java code we use [this](https://github.com/mapbox/mapbox-navigation-android/tree/master/config/checkstyle) checkstyle and code style. You can find instructions on how to set both in [Setting up Mapbox checkstyle](https://github.com/mapbox/mapbox-gl-native-android/wiki/Setting-up-Mapbox-checkstyle). The same thing happens 👇

Reply:
it has been already done, so this style is applied by checkstyle from config *folder*

Reply:
has applied Squad android-java codestyle